event/src/app/api/events/[id]/route.ts

```diff
@@ -0,0 +1,69 @@
+import { IUserPayload } from "@/app/api/users/login/route";
+import { NextRequest, NextResponse } from "next/server";
+import * as jose from 'jose'
+import Event, { Events } from "@/models/Event";
+import dbConnect from "@/lib/mongo";
+import { EventSchema, IEvent } from "../host/route";
```

Comment:
Consider moving `EventSchema` and `IEvent` to a different file, maybe `@/lib/types` or `@/models/Event`